packages/samples/test/output/visibility/openapi.yaml

```diff
@@ -236,10 +233,6 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/PersonRelativeUpdateItem'
-      required:
-        - name
-        - other
-        - relatives
```

Comment:
But this does seem correct. If these are updatable properties, they should be optional, not required, yes?

Reply:
This is the same as above, array elements should keep their required fields.

Reply:
Fixed.